### DISCUSSION TITLE
Is it necessary for FluxCD to have access to Kubernetes nodes root directories?

### DISCUSSION THREAD

**Discussion by rajagopalrao** [fluxcd/flux2#4690] [Q&A]
Is it necessary for FluxCD to have access to Kubernetes nodes root directories? If so, then how does FluxCD tackle the Security policies that are applied to Kubernetes Clusters?

**Comment by stefanprodan** [comment]
Answered here https://github.com/fluxcd/flux2/issues/4691#issuecomment-2018061948
